classes/ImageManager.php

```diff
@@ -258,7 +266,6 @@ public static function resize($src_file, $dst_file, $dst_width = null, $dst_heig
             ImageManager::imagecopyresampled($dest_image, $src_image, (int)(($dst_width - $next_width) / 2), (int)(($dst_height - $next_height) / 2), 0, 0, $next_width, $next_height, $src_width, $src_height, $quality);
         }
         $write_file = ImageManager::write($file_type, $dest_image, $dst_file);
-        Hook::exec('actionOnImageResizeAfter', array('dst_file' => $dst_file, 'file_type' => $file_type));
```

Comment:
Hi,

I don't. Well I do but I don't want. I'm learning git and I get better 
times after times.

I will pull again.

Regards and sorry, Edouard

Le 26/05/2016 23:52, Mickaël Andrieu a écrit :
>
> In classes/ImageManager.php 
> <https://github.com/PrestaShop/PrestaShop/pull/5648#discussion_r64827636>:
>
> > @@ -258,7 +266,6 @@ public static function resize($src_file, $dst_file, $dst_width = null, $dst_heig
> >              ImageManager::imagecopyresampled($dest_image, $src_image, (int)(($dst_width - $next_width) / 2), (int)(($dst_height - $next_height) / 2), 0, 0, $next_width, $next_height, $src_width, $src_height, $quality);
> >          }
> >          $write_file = ImageManager::write($file_type, $dest_image, $dst_file);
> > -        Hook::exec('actionOnImageResizeAfter', array('dst_file' => $dst_file, 'file_type' => $file_type));
>
> why do you remove hooks ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/PrestaShop/PrestaShop/pull/5648/files/abfee998087dc64cff18a879c76261a45df68998#r64827636>
>